packages/opc-back-to-top/README.md

```diff
@@ -0,0 +1,56 @@
+# This is opc-back-to-top 👋
+
+A standardized web component based on Lit element for Red Hat One Platform back to top
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+### Back-to-top with revealAt attribute:
+<opc-back-to-top revealAt="200"></opc-back-to-top>
+
+### Back-to-top without revealAt attribute:
+<opc-back-to-top></opc-back-to-top>
```

Comment:
Can you add more information... please check the readme of the header and footer component to take reference.

Reply:
documentation is missing on the usage in angular and react.